po/de_DE.po

```diff
@@ -0,0 +1,706 @@
+msgstr "‚{}‘ konnte nicht entfernt werden"
+
+#: src/clean.rs:93
+msgid "can't open clone dir: {}"
+msgstr "Clone-Verzeichnis kann nicht geöffnet werden: {}"
+
+#: src/command_line.rs:127
+msgid "option {} expects a value"
+msgstr "Option {} erwartet einen Wert"
+
+#: src/command_line.rs:158 src/command_line.rs:327
+msgid "option {} does not allow a value"
+msgstr "Option {} erlaubt keinen Wert"
+
+#: src/command_line.rs:318
+msgid "unknown option --{}"
+msgstr "Unbekannte Option --{}"
+
+#: src/command_line.rs:321
+msgid "unknown option -{}"
+msgstr "Unbekannte Option -{}"
+
+#: src/completion.rs:27
+msgid "failed to open cache file '{}'"
+msgstr "Cache-Datei ‚{}‘ konnte nicht geöffnet werden"
+
+#: src/completion.rs:65
+msgid "could not update aur cache"
+msgstr "AUR-Cache konnte nicht geupdated werden"
+
+#: src/config.rs:174
+msgid "invalid value '{:val}' for key '{:key}', expected: {:exp}"
+msgstr "Ungültiger Wert ‚{:val}' für Schlüssel ‚{:key}', erwartet: {:exp}"
+
+#: src/config.rs:474
+msgid "failed to find cache directory"
+msgstr "Cache-Verzeichnis konnte nicht gefunden werden"
+
+#: src/config.rs:477
+msgid "failed to find config directory"
+msgstr "Konfigurations-Verzeichnis konnte nicht gefunden werden"
+
+#: src/config.rs:501
+msgid "config file '{}' does not exist"
+msgstr "Konfigurations-Datei ‚{}‘ existiert nicht"
+
+#: src/config.rs:550
+msgid "only one operation may be used at a time"
+msgstr "Es kann nur eine Operation gleichzeitig ausgeführt werden"
+
+#: src/config.rs:638
+msgid "can not find local repo '{}' in pacman.conf"
+msgstr "Lokales Repository ‚{}‘ in pacman.conf kann nicht gefunden werden"
+
+#: src/config.rs:670
+msgid "failed to initialize alpm: root={} dbpath={}"
+msgstr "alpm konnte nicht initialisiert werden: root={} dbpath={}"
+
+#: src/config.rs:740 src/config.rs:873
+msgid "value can not be empty for key '{}'"
+msgstr "Der Wert für den Schlüssel ‚{}‘ darf nicht leer sein"
+
+#: src/config.rs:756
+msgid "key '{}' does not belong to a section"
+msgstr "Der Schlüssel ‚{}‘ gehört zu keinem Abschnitt"
+
+#: src/config.rs:762
+msgid "unknown section '{}'"
+msgstr "Unbekannter Abschnitt ‚{}‘"
+
+#: src/config.rs:769 src/config.rs:824 src/config.rs:829
+msgid "key can not be empty"
+msgstr "Der Schlüssel darf nicht leer sein"
+
+#: src/config.rs:791
+msgid "error: unknown option '{}' in section [bin]"
+msgstr "Fehler: Unbekannte Option ‚{}‘ in Abschnitt [bin]"
+
+#: src/config.rs:891
+msgid "error: unknown option '{}' in section [options]"
+msgstr "Fehler: Unbekannte Option ‚{}‘ in Abschnitt [options]"
+
+#: src/config.rs:896
+msgid "option '{}' does not take a value"
+msgstr "Option ‚{}‘ nimmt keinen Wert"
+
+#: src/config.rs:947 src/install.rs:1796
+msgid "There are {:n} providers available for {:pkg}:"
+msgstr "Es sind {:n} Anbieter für {:pkg} verfügbar:"
+
+#: src/config.rs:961 src/info.rs:56 src/info.rs:95 src/install.rs:1755
+#: src/install.rs:1804
+msgid "Repository"
+msgstr "Repository"
```

Comment:
In the [pacman repo,](https://gitlab.archlinux.org/pacman/pacman/-/blob/master/src/pacman/po/de.po) "Repository" translates to "Repositorium". Sounds unnatural, but might be worth a thought.